packages/terra-core-docs/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+* Added
+  * Added documentation updates for `Single Select`.
+
+* Changed
+  * Removed aria-checked to make screen readers announce Selected in `terra-form-select`.
+  * Updated focus to shift back on select upon key press or mouse click event while using Safari browser in `terra-form-select`.
```

Comment:
Since these changes are from `terra-form-select`, they do not need to be included in the `terra-core-docs` CHANGELOG.